Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MessageKit",
+    platforms: [.iOS(.v11)],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(name: "MessageKit", targets: ["MessageKit"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/classdojo/InputBarAccessoryView", .branch("dropRxSPM"))
```

Comment:
Linking to fork without RxSwift dependency while waiting on merging this to `InputBarAccessoryView`